aten/src/ATen/native/native_functions.yaml

```diff
@@ -197,36 +197,36 @@
 - func: addr_(Tensor(a!) self, Tensor vec1, Tensor vec2, *, Scalar beta=1, Scalar alpha=1) -> Tensor(a!)
   variants: method
 
-- func: addr(Tensor self, Tensor vec1, Tensor vec2, *, Scalar beta=1, Scalar alpha=1, Tensor(a!) out) -> Tensor(a!)
+- func: addr.out(Tensor self, Tensor vec1, Tensor vec2, *, Scalar beta=1, Scalar alpha=1, Tensor(a!) out) -> Tensor(a!)
 
 - func: affine_grid_generator(Tensor theta, int[] size) -> Tensor
   variants: function
 
 - func: affine_grid_generator_backward(Tensor grad, int[] size) -> Tensor
   variants: function
 
-- func: all(Tensor self, int dim, bool keepdim=False) -> Tensor
+- func: all.dim(Tensor self, int dim, bool keepdim=False) -> Tensor
   variants: function, method
 
-- func: all(Tensor self, int dim, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
+- func: all.out(Tensor self, int dim, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
 
 - func: allclose(Tensor self, Tensor other, float rtol=1e-05, float atol=1e-08, bool equal_nan=False) -> bool
   variants: function, method
 
-- func: any(Tensor self, int dim, bool keepdim=False) -> Tensor
+- func: any.dim(Tensor self, int dim, bool keepdim=False) -> Tensor
   variants: function, method
 
-- func: any(Tensor self, int dim, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
+- func: any.out(Tensor self, int dim, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
 
 - func: arange(Scalar end, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
 
-- func: arange(Scalar start, Scalar end, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
+- func: arange.start(Scalar start, Scalar end, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
 
-- func: arange(Scalar start, Scalar end, Scalar step, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
+- func: arange.start_step(Scalar start, Scalar end, Scalar step, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
 
-- func: arange(Scalar end, *, Tensor(a!) out) -> Tensor(a!)
+- func: arange.out(Scalar end, *, Tensor(a!) out) -> Tensor(a!)
 
-- func: arange(Scalar start, Scalar end, Scalar step=1, *, Tensor(a!) out) -> Tensor(a!)
+- func: arange.out_start(Scalar start, Scalar end, Scalar step=1, *, Tensor(a!) out) -> Tensor(a!)
```

Comment:
For consistency with the other rules, it probably makes more sense for it to be `start_out` rather than `out_start`.